tools/sgen/input/FluentUIStyle.yml

```diff
@@ -161,3 +163,21 @@ SecondaryButtonTokens extends MSFButtonTokens:
 GhostButtonTokens extends MSFButtonTokens:
   borderColor: [ rest: $Colors.Neutral.clear, hover: $Colors.Neutral.clear, pressed: $Colors.Neutral.clear, selected: $Colors.Brand.shade10, disabled: $Colors.Neutral.clear ]
 
+AP_MSFListTokens:
+  backgroundColor: [ rest: $Colors.Background.neutral1, pressed: $Colors.Background.neutral5 ]
+  borderColor: $Colors.Stroke.neutral1
+  borderSize: $Border.size.thin
+  disclosureIconForegroundColor: $Colors.Foreground.neutral3
+  disclosureInterspace: $Spacing.xxSmall
+  disclosureSize: $Icon.size.small
+  horizontalCellPadding: $Spacing.medium
+  iconColor: $Colors.Foreground.neutral4
+  iconInterspace: $Spacing.medium
+  iconSize: [ icon: $Icon.size.medium, largeIcon: $Icon.size.xxlarge ]
+  labelColor: $Colors.Foreground.neutral1
+  sublabelColor: $Colors.Foreground.neutral3
+  textFont: [ label: $Typography.body, subLabel: $Typography.caption1 ]
```

Comment:
I think this can be split into 2 tokens:
labelTextFont: $Typography.body
subLabelTextFont: $Typography.caption1